ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDIndividual.java

```diff
@@ -1,10 +1,12 @@
 package uk.gov.hmcts.cmc.ccd.domain;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.Builder;
 import lombok.Value;
 
 @Value
 @Builder
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
```

Comment:
Would that make sense to configure it per application instead of adding it to all domain classes?